doc-content/enterprise-only/migration/migration-configure-kie-server-proc.adoc

```diff
@@ -58,3 +58,32 @@ If the {CENTRAL} port is 8080, as in this example, then the {KIE_SERVER} port, w
 ====
 
 {KIE_SERVER} connects to the new {CENTRAL} and collects the list of deployment units (KIE containers) to be deployed.
+
+
+[NOTE]
+====
+When you use a class inside a dependency JAR file to access {KIE_SERVER} from {KIE_SERVER} client, you get the `ConversionException` and `ForbiddenClassException` in {CENTRAL}.
+To avoid generating these exceptions in {CENTRAL}, do one of the following:
+
+* If the exceptions are generated on the client-side, add following system property to the kie-server client:
+[source]
+----
+System.setProperty("org.kie.server.xstream.enabled.packages", "org.example.**");
+----
+* If the exceptions are generated on the server-side, open `standalone-full.xml` from the {PRODUCT} installation directory, set the following property under the <system-properties> tag:
+[source]
+----
+<property name="org.kie.server.xstream.enabled.packages" value="org.example.**"/>
+----
+* Set the following JVM property:
+[source]
+----
+-Dorg.kie.server.xstream.enabled.packages=org.example.**
+----
+It is expected that you do not configure the classes that exits in KJAR using these system property. Ensure that only known classes are used in the system property to avoid any vulnerabilities.
+
+The `org.example` package is mandatory. You can specify multiple packages separated by comma , for example,
+`org.example1.* * , org.example2.* * , org.example3.* *`.
+
+You can also add specific classes , for example, `org.example1.* * , org.example2.Mydata2`.
```

Comment:
Should be perhaps `org.example1.Mydata1`?

Reply:
Done